### PR TITLE
#334 Rename medicinal_product toggle

### LINKED ISSUE #334: Buttons on UCS web
Migrated from https://github.com/sussol/universal-drug-codes-server/issues/62

Drug = medicinal product?

I think that drug should be medicinal product.  Medicinal product should be consumables?

I think we were also going to have another but that might not be needed with the asset management module (or at least not needed now).

### PR DESCRIPTION
**Pull request by katherine-sussol**
Fixes #334 

## Description

Nothing really major, just renaming a toggle button to match what it's actually retrieving

### Documentation:

Tick below if one of the following applies:

1. Documentation has been added to cover changes introduced by this PR.
2. This PR requires no changes to documentation.

- [x] Documentation OK.

### Tests:

Tick below if one of the following applies:

1. Tests have been added to cover changes introduced by this PR.
2. This PR requires no new tests.

- [x] Tests OK.

### CODE REVIEW

File: unified-codes/libs/data/src/lib/v1/Entity.ts [hunk 1]
```diff
@@ -2,7 +2,7 @@ import { IProperty, Property } from './Property';
 
 export enum EEntityCategory {
   DRUG = 'drug',
-  MEDICINAL_PRODUCT = 'medicinal_product',
+  CONSUMABLES = 'consumables',
```
Comment (wlthomson) [inline unified-codes/libs/data/src/lib/v1/Entity.ts:5]: For consistency, may as well just call this category `CONSUMABLE` (can leave the label as `"Consumables"`)?

**Review by wlthomson — APPROVED** [review]
Minor naming nit. Apart from that:

giphy

File: unified-codes/libs/data/src/lib/v1/Entity.ts (hunk 1 above)
Comment (katherine-sussol) [reply unified-codes/libs/data/src/lib/v1/Entity.ts]: Good idea! e9d8d341b

**Comment by wlthomson** [comment]
Conflicts 😢 .

**Comment by katherine-sussol** [comment]
> Conflicts 😢 .

😿  will have a look

**Pull request merged**